xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/uicomponents/viewers/comments.js

```diff
@@ -65,69 +64,6 @@ viewers.Comments = Class.create({
       item._x_number = elementId.substring(elementId.lastIndexOf("_") + 1) - 0;
     });
   },
-  /**
-   * Ajax comment deletion.
-   * For all delete buttons, listen to "click", and make ajax request to remove the comment. Remove the corresponding
-   * HTML element on succes (replace it with a small notification message). Display error message (alert) on failure.
-   */
-  addDeleteListener : function() {
-    $$(this.xcommentSelector).each(function(item) {
-      // Prototype bug in Opera: $$(".comment a.delete") returns only the first result.
-      // Quick fix until Prototype 1.6.1 is integrated.
-      item = item.down('a.delete');
-      if (!item) {
-        return;
-      }
-      item.observe('click', function(event) {
-        item.blur();
-        event.stop();
-        if (item.disabled) {
-          // Do nothing if the button was already clicked and it's waiting for a response from the server.
-          return;
-        } else {
-          new XWiki.widgets.ConfirmedAjaxRequest(
-            /* Ajax request URL */
-            item.readAttribute('href') + (Prototype.Browser.Opera ? "" : "&ajax=1"),
-            /* Ajax request parameters */
-            {
-              onCreate : function() {
-                // Disable the button, to avoid a cascade of clicks from impatient users
-                item.disabled = true;
-              },
-              onSuccess : function() {
-                // Remove the corresponding HTML element from the UI and update the comment count
-                var comment = item.up(this.xcommentSelector);
-                // If the form is inside this comment's reply thread, move it back to the bottom.
-                if (this.form && this.form.descendantOf(comment.next('.commentthread'))) {
-                  this.resetForm();
```

Comment:
You removed this lines but I don't see anything to replace them. What happens when the reply form is inside the comment thread and we remove the comment? Do we loose the form?